scripts/commands/getstats.lua

```diff
@@ -4,14 +4,22 @@
 -----------------------------------
 local commandObj = {}
 
+local options =
+{
+    'base',
+    'defensive',
+    'offensive',
+    'elements',
+}
+
 commandObj.cmdprops =
 {
     permission = 1,
-    parameters = ''
+    parameters = 'i'
 }
 
 -- function onTrigger(player, extendedMode)
```

Comment:
this line is a remnant of my original intention to have a string param for extended modes (spit out params other than the default) and can now be removed

Reply:
Removed